BorrowToOwn.API/BorrowToOwn.Migrations/Data/20200628211908_postgres full-text search trigger.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BorrowToOwn.API.BorrowToOwn.Migrations.Data
+{
+    public partial class postgresfulltextsearchtrigger : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            //            migrationBuilder.Sql(@"CREATE PROCEDURE SelectAllCustomers AS SELECT * FROM ""Products"" ");
+            //            migrationBuilder.Sql(@"CREATE TABLE ""Testing"" (
+            //    TestingID int,
+            //    LastName varchar(255),
+            //    FirstName varchar(255),
```

Comment:
i'll merge but it will be nice to remove these comments

Reply:
yeah ... on it. Thanks